examples/kitchen-sink/tina/config.tsx

```diff
@@ -16,27 +16,26 @@ const router = ({ document, collection }) => {
 const extendedRouter = ({ document, collection }) => {
   return `/${collection.name}/${document._sys.breadcrumbs.join('/')}`
 }
+
+const customAuthProvider = {
+  getToken: async () => {
+    return { id_token: 'some-token' }
+  },
+  logout: async () => {
+    localStorage.removeItem(TINA_TOKEN_KEY)
+  },
+  authenticate: async () => {
+    localStorage.setItem(TINA_TOKEN_KEY, 'some-token')
+    return true
+  },
+  getUser: async () => {
+    return localStorage.getItem(TINA_TOKEN_KEY)
+  },
+}
 export default defineConfig({
   // contentApiUrlOverride: '/api/gql',
   admin: {
-    auth: {
-      useLocalAuth: true,
-      // If you wanted to use custom auth
-      customAuth: true,
-      getToken: async () => {
-        return { id_token: 'some-token' }
-      },
-      logout: async () => {
-        localStorage.removeItem(TINA_TOKEN_KEY)
-      },
-      authenticate: async () => {
-        localStorage.setItem(TINA_TOKEN_KEY, 'some-token')
-        return true
-      },
-      getUser: async () => {
-        return localStorage.getItem(TINA_TOKEN_KEY)
-      },
-    },
+    auth: customAuthProvider,
```

Comment:
is this correct?

Reply:
I think this could be removed and it would work the same.

Passing an object to the admin provider is correct but a class is preferred. 

Should we remove he customAuthProvider from this example?

Reply:
I think its fine as-is if it works - maybe better to set it via authProvider since admin.auth is deprecated, though